package.json

```diff
@@ -71,18 +72,23 @@
     "defined": "1.0.0",
     "express": "4.14.0",
     "history": "2.1.2",
+    "inuitcss": "^6.0.0-beta.3",
     "isomorphic-fetch": "2.2.1",
     "lodash": "4.15.0",
+    "moment": "2.15.1",
     "ndla-ui": "0.0.3",
     "node-uuid": "1.4.7",
     "normalize-css": "2.3.1",
     "react": "15.3.1",
+    "react-copy-to-clipboard": "4.2.3",
     "react-dom": "15.3.1",
     "react-helmet": "3.1.0",
     "react-intl": "2.1.5",
+    "react-magic-effects": "^0.1.0",
```

Comment:
dev junk. Men beholder kanskje react-copy-to-clipboard for siteringsfunksjonen. Men så nøler jeg litt med å introdusere masse greier så tidlig.